abcfold/af3_script_utils.py

```diff
@@ -282,12 +283,11 @@ def make_dir(dir_path: Union[str, Path], overwrite: bool = False):
     dir_path = Path(dir_path)
     if dir_path.exists():
         if overwrite:
-            dir_path.rmdir()
+            shutil.rmtree(dir_path)
```

Comment:
Good change! I noticed overwrite wasn't working the other day and meant to mention it